packages/react-core/src/components/Card/examples/CardSelectable.tsx

```diff
@@ -6,6 +6,7 @@ export const SelectableCard: React.FunctionComponent = () => {
   const [isChecked2, setIsChecked2] = useState(false);
   const [isChecked3, setIsChecked3] = useState(false);
   const [isSecondary, setIsSecondary] = useState<boolean>(false);
```

Comment:
We can omit the explicit types in these hooks since the initial value is a boolean which TS infers.

```suggestion
  const [isSecondary, setIsSecondary] = useState(false);
```